src/components/sign-in/sign-in.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { loginSuccess } from '../../redux/actions/session';
+import './sign-in.styles.scss';
+
+class SignIn extends React.Component {
+	constructor() {
+		super();
+		this.state = {
+			email: '',
+			password: ''
+		}
+	}
+
+	handleChange = event => {
+		const { name, value } = event.target;
+		this.setState({ [name]: value });
+	}
+
+	submitForm = (event) => {
```

Comment:
Some things to note:

1. This isn't the right place to do the data fetching. 
2. Data fetching/API communicating logic should be as an asynchronous **action** (part of the actions file).
3. I truly suggest to simplify our project first (make sure we first do all Milestone 1 to 5 related stuff only for now).